docs/config/README.md

```diff
@@ -1142,19 +1142,24 @@ The module will be shown if any of the following conditions are met:
 
 ### Options
 
-| Variable   | Default      | Description                                            |
-| ---------- | ------------ | ------------------------------------------------------ |
-| `symbol`   | `"💎 "`      | The symbol used before displaying the version of Ruby. |
-| `style`    | `"bold red"` | The style for the module.                              |
-| `disabled` | `false`      | Disables the `ruby` module.                            |
+| Option     | Default                          | Description                 |
+| ---------- | -------------------------------- | --------------------------- |
+| `format`   | `"via [💎 $version](red bold) "` | The format for the module.  |
+| `disabled` | `false`                          | Disables the `ruby` module. |
```

Comment:
We are keeping the `symbol` and `style` options to make the migration easier can you add them back please.